### ISSUE TITLE
Accessibility - Autdit

### ISSUE THREAD

**Issue by BraedonB98** [BraedonB98/RoadRunner-Resource-Hub-Frontend#17]
Do Research into how to audit your own website for accessibility, then audit the website, add "failed accessibility" to the project board
